pagopa/io-app
# fix: [#177001144] Changed latest transactions label when transactions are hidden

## Short description
Now the style of "Latest transactions" label when transactions are hidden is equal to "Latest transactions" label when transactions are visible

## How to test
Move into wallet screen, click show transactions button and see if the style of "Latest transactions" label is always the same

**Before changes:**
<img width="300" alt="Screenshot 2021-04-26 at 13 02 28" src="https://user-images.githubusercontent.com/61834253/116074045-7d71a280-a691-11eb-90eb-56e115062344.png">

**After changes:**
<img width="300" alt="Screenshot 2021-04-26 at 13 16 04" src="https://user-images.githubusercontent.com/61834253/116074178-a8f48d00-a691-11eb-9bad-0b193c122045.png">

<img width="300" alt="Screenshot 2021-04-26 at 14 34 47" src="https://user-images.githubusercontent.com/61834253/116083289-bf541600-a69c-11eb-8241-08ee74458e52.png">
